snprc_ehr/resources/queries/study/availableChargeIds.sql

```diff
@@ -0,0 +1,62 @@
+PARAMETERS (
+   	ANIMAL_ID VARCHAR,
+    EVENT_DATE TIMESTAMP
+)
+
+-- Research Charge IDs
+SELECT
+    CAST(p.project AS VARCHAR(40)) AS admitProjectId,
+    p.project AS projectId,
+    sp.projectType AS projectType,
+    0 AS admitId,
+    sp.description AS projectText,
+    GREATEST (p.startDate, aaa.date, sp.startDate) AS startDate,
+    LEAST (p.endDate, TIMESTAMPADD('SQL_TSI_DAY', 1, aaa.endDate), TIMESTAMPADD('SQL_TSI_DAY', 1, sp.endDate), NOW()) AS endDate,
```

Comment:
Can i please know what is this "SQL_TSI_DAY"? just for my understanding.

Reply:
It tells the timestampadd function to use days as the unit to add to a date. Check out the LK SQL Reference: https://www.labkey.org/Documentation/wiki-page.view?name=labkeysql